.github/workflows/gating.yaml

```diff
@@ -8,6 +8,15 @@ on:
   workflow_dispatch:
     inputs: {}
 
+env:
+  IMAGE_NAME: cachi2
+  IMAGE_REGISTRY: quay.io
+  IMAGE_NAMESPACE: containerbuildsystem
+  IMAGE_TAG: latest
```

Comment:
should we differentiate between PR and main branch builds?

Reply:
Not sure if I get this comment. Currently, we build images locally in open PR, but we do not push them anywhere. Only merged PRs will update the cachi2:latest image in quay.io . Can you please elaborate more here?

Reply:
ah sorry, I haven't noticed the condition in task